### PR TITLE
Infinite scroll change

### PR DESCRIPTION
**Pull request by Pitros**
Updated infinite scroll and moved scrollbar to mainView for nicer look.

### CODE REVIEW

**Comment by weblancaster** [comment]
Great! thanks @Pitros I'm going to review today or tomorrow.

**Comment by weblancaster** [comment]
@Pitros This PR actually looks like introduced a bug.. I did not see any delay in the infinite scroll but testing this one I noticed that the scroll has a delay and give the impression that new tracks on load.

Can you tell me the bug you found in the infinite-scroll to add this fix please?

**Comment by Pitros** [comment]
Bug was with `infinite-scroll-disabled='{{ busy }}'`, it just crashes whole app, I removed it in current PR.
Delay 1,5s is probably is caused by this line: 
```javascript
angular.module('infinite-scroll').value('THROTTLE_MILLISECONDS', 1500)
```
It might not work on older version of infinite-scroll or that's not an issue and delay is caused by removing infinite-scroll-disabled attribute (but kinda has no sense for me).

File: app/public/stylesheets/sass/_components/_view-container.scss [hunk 1]
```diff
@@ -2,6 +2,8 @@
   // flex: 5 0 0;
   padding: 10px 20px;
   margin-left: 200px;
+  overflow-y: scroll;
```
Comment (weblancaster) [inline app/public/stylesheets/sass/_components/_view-container.scss]: can you add `overflow-x: hidden` please.

**Comment by weblancaster** [comment]
@Pitros yeah still has a little of delay... but I think looks good though.

Can you update the `angular.module('infinite-scroll').value('THROTTLE_MILLISECONDS', 200)` and the overflow comment I did.. then it's good to go.

**Comment by Pitros** [comment]
Pushed new files, check if everything is ok.

**Pull request merged**